qutip/qip/gates.py

```diff
@@ -918,8 +922,8 @@ def gate_expand_3toN(U, N, controls=[0, 1], target=2):
         raise ValueError(
             "control and not target is None must be integer < integer N")
 
-    if (controls[0] == target or controls[1] == target
-            or controls[0] == controls[1]):
+    if controls[0] == target or controls[1] == target \
```

Comment:
Is this better practice? I often use brackets to avoid the use of \
just cause I don't like it :)